_pages/security/scanning.md

```diff
@@ -21,3 +21,41 @@ Alerts come in one of four categorizations:
 * **High** and **Critical** must be resolved before the ATO is granted.
 
 Any false positives should be documented as such, through an "ignore" file (e.g. with a `note` in [Brakeman](http://brakemanscanner.org/docs/ignoring_false_positives/)) or with your ATO-related documentation.
+
+## ATO Scanning 
+
+### Getting ready to scan
+
+The system's technical stack needs to be relatively stable before authorization.
+
+If, during testing, the system performs:
+
+* user authentication or authorization
+* back-end administrative functions
+* encryption
+
+...then those features cannot be "relatively simple" but must in fact be *complete* before an authorization will be given. Note that the use of common web frameworks and [18F HTTPS standards](https://github.com/18F/https) resolves these issues in almost every case.
+
+Lastly, make sure the `README` file in your repo is fully up to date and clearly explains what the system does and why at a high level. You should also include the above information in your `README`.
```

Comment:
I guess I mean that the "Getting ready to scan" information isn't specific to scanning. It's all _relevant_ for scanning (especially to give context for anyone outside of the project team doing scanning), but is more broadly applicable (for the security documentation, etc.).

Happy to punt on this conversation so it doesn't hold up all the other changes in this pull request.